x-pack/legacy/plugins/maps/public/layers/styles/vector_style_defaults.js

```diff
@@ -29,20 +33,22 @@ export function getDefaultProperties(mapColors = []) {
 export function getDefaultStaticProperties(mapColors = []) {
   // Colors must be state-aware to reduce unnecessary incrementation
   const lastColor = mapColors.pop();
-  const nextColorIndex = (DEFAULT_COLORS.indexOf(lastColor) + 1) % (DEFAULT_COLORS.length - 1);
-  const nextColor = DEFAULT_COLORS[nextColorIndex];
+  const nextColorIndex = (DEFAULT_FILL_COLORS.indexOf(lastColor) + 1) % (DEFAULT_FILL_COLORS.length - 1);
```

Comment:
There is an existing bug in this function where the last color in the list is never provided as an option. Removing the `- 1` resolves this problem so then all colors a available.

`const nextColorIndex = (DEFAULT_FILL_COLORS.indexOf(lastColor) + 1) % (DEFAULT_FILL_COLORS.length);`